rgeo_test.go

```diff
@@ -122,6 +122,7 @@ var testdata = []struct {
 			SubRegion:    "Northern America",
 			Province:     "Alaska",
 			ProvinceCode: "US-AK",
+			County:       "", // unknown
```

Comment:
https://github.com/sams96/rgeo/pull/28#issuecomment-1847691009
That's why this is empty. Here, `TYPE = Municipality`.